README.md

```diff
@@ -95,6 +95,20 @@ chatkit_integration_locator=<instance locator>
 chatkit_integration_key=<instance key>
 ```
 
+### Spek Tests
+
+Our tests are currently written using the [Spek Framework](https://www.spekframework.org/setup-android/).
+And you will need to install the Spek Plugin in Android Studio to be able to run them.
+
+You may find that you will also need to edit the run configuration and add your
```

Comment:
I would make it explicit as it's not "you may find" but "you will definitely need". That point is also already covered above. I would just make it more explicit like it is below and remove `local.properties` stuff as I don't think that ever worked as that file is just read by Android Gradle Plugin (not Gradle).

Reply:
Gotcha! I didn't want to change too much what was already there in case it worked for others. But you're probably right, the other one probably didn't work for anyone else :)